server.js

```diff
@@ -155,4 +156,5 @@ MongoClient.connect(config.db, function(err, db) {
     });
     */
 
+    opn("http://localhost:" + config.port);
```

Comment:
@petermbenjamin Thanks for the PR. I see the convenience of automatically opening the app in the browser on starting the server. However, I am a bit unsure how this would work when the app is hosted on services such as heroku, especially as the `opn` command has localhost hardcoded. We can possibly add a logic to conditionally execute the `opn` command only if the hostname is localhost. Any thoughts?